pylibczi/CziFile.py

```diff
@@ -134,9 +199,14 @@ def convert_to_buffer(file: types.FileLike) -> Union[BinaryIO, np.ndarray]:
             )
 
     def read_meta(self):
```

Comment:
I would change this to a property that reads once then stores like it does in AICSImage